README.md

```diff
@@ -17,14 +17,11 @@ The provider comes in two parts:
 
 ## Installation
 
-> [!CAUTION]
-> This plugin is not ready for general use and is awaiting changes to be merged in yt-dlp for it to be functional.
-> Follow https://github.com/yt-dlp/yt-dlp/pull/10648 for updates.
-
 ### Base Requirements
 
-If using Docker image for option (a) for the provider, the Docker runtime is required.
+1. **Requires yt-dlp `MASTER 2024.09.13.105220` or above.**
```

Comment:
Let's keep the [!CAUTION] for now until its in stable channel

Reply:
then maybe just merge this pr later. remember to change the potoken docs link to `_[What is a PO Token?](https://github.com/yt-dlp/yt-dlp/wiki/Extractors#po-token-guide)_` though

Reply:
We could still merge it now because mention that they need to be on nightly channel instead of stable. Not sure how long it will take until its in stable
https://github.com/yt-dlp/yt-dlp/actions/runs/10857446502

Reply:
```suggestion
1. **Requires yt-dlp `NIGHTLY 2024.09.13.232912` or above.**
```